### PR TITLE
docs: added examples for p5.Element.class

### PR DESCRIPTION
**Pull request by austince**
Adding two examples for `p5.Element.class` as listed here: #1954 🎊

Perhaps note in `pressed` doesn't belong here but I thought it was a nice way to show a way the zero-args overload was useful. Could make it more useful with `add/removeClass`

### CODE REVIEW

**Comment by austince** [comment]
Having some trouble with the linter. Can't find task `eslint-samples:source` using the grunt-cli

File: src/core/p5.Element.js [hunk 1]
```diff
@@ -145,7 +145,42 @@ p5.Element.prototype.id = function(id) {
...
+ * function setup() {
+ *   const personText = createP("A person's a person");
+ *   personText.class('seuss');
+ *   createP('No matter how small.').class('seuss');
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * no display.
```
Comment (lmccart) [inline src/core/p5.Element.js]: can you please add a text description of what is visible in the example?

File: src/core/p5.Element.js [hunk 2]
```diff
@@ -145,7 +145,42 @@ p5.Element.prototype.id = function(id) {
...
+ * function pressed() {
+ *  // `this` refers to the clicked element.
+ *  // Sets the class name to the pressed element's name plus '-pressed'
+ *  this.class(this.class() + '-pressed');
+ * }
+ * </code>
+ * </div>
+ *
+ * @alt
+ * no display.
```
Comment (lmccart) [inline src/core/p5.Element.js]: can you please add a text description of what is visible in the example, and what changes with mouse interaction? this makes the example accessible for seeing impaired users.

**Comment by lmccart** [comment]
we're all set for an example for this one now, thanks!

**Pull request closed without merging**